pkg/ingester/errors_test.go

```diff
@@ -237,6 +237,18 @@ func TestNewTSDBIngestExemplarErr(t *testing.T) {
 	checkIngesterError(t, wrappedErr, mimirpb.BAD_DATA, true)
 }
 
+func TestTooBusyError(t *testing.T) {
+	require.Error(t, tooBusyError)
+	require.Equal(t, "the ingester is currently too busy to process queries, try again later", tooBusyError.Error())
```

Comment:
[Nit] Simplification
```suggestion
	require.EqualError(t, tooBusyError, "the ingester is currently too busy to process queries, try again later")
```